templates/googleapis_commit.txt

```diff
@@ -0,0 +1 @@
+dc5611c6820826fb46ca2ca1399cf8dc5b32f600
```

Comment:
This is the change in googleapis 2 hours ago

https://github.com/googleapis/googleapis/commit/dc5611c6820826fb46ca2ca1399cf8dc5b32f600